src/components/Timeline/Event.js

```diff
@@ -20,13 +21,16 @@ export const Event = ({
     eventTypeIsFetching,
     eventTypeIsError,
     eventId,
-    event
+    event,
+    actions
 }) => {
     const itemHighlight = (event && event.timeReceived) ? {
         animationName: 'yellowfade',
         animationDuration: '30s',
         animationDelay: -(moment().diff(event.timeReceived, 'seconds')) + 's'
     } : {}
+    const isAllPlaybookInfoAvailable = actions.length > 0
```

Comment:
I think this should be renamed to `isPlaybookAvailable` (I don't believe this test case covers _all_, but instead _any_ playbook availability, and info is meaningless.)